src/commands/test.ts

```diff
@@ -387,11 +388,26 @@ async function processTestScripts(
         }
 
         console.log(`Run test function ${test}`);
-        await testSuiteInstance[test]();
-        if (expectedRevertMessage !== undefined) {
-          throw new Error(
-            `Test was expected to revert with message "${expectedRevertMessage}" but no revert happened.`
-          );
+
+        const noOfFuzzingParameters = testSuiteInstance[test].length;
+        const testIterations = noOfFuzzingParameters > 0 ? 64 * noOfFuzzingParameters : 1;
```

Comment:
I think it would be better to extract the `64` into a variable or some config parameter. Otherwise it's not clear what it means and why it is 64.

Reply:
Put into a constant called `NUMBER_OF_FUZZING_ITERATIONS` for now.

Later we should make this more flexible and configurable.